cmd/daemon/command/start.go

```diff
@@ -34,6 +34,22 @@ func StartDaemon() *cobra.Command {
 			logConfiguration.ParseFromEnvironmnet()
 			log.Init(logConfiguration)
 
+			idpURL := os.Getenv("HAMCTL_OAUTH_IDP_URL")
```

Comment:
Per conversation: This should be flags instead and the hamctl part should be removed.